doc/articles/uno-development/guidelines-for-creating-and-triaging-issues.md

```diff
@@ -0,0 +1,191 @@
+
+## Support requests
+
+These issues should be labeled with `triage/support`, directed to our support structures (see below) and then closed _but_ before doing so consider why the support request was raised.
+
+Is something unclear? Do we need to improve our documentation or samples?
+
+Community support requests should be directed to:
+
+* Our [documentation](https://platform.uno/docs/) and [troubleshooting guide](https://platform.uno/docs/troubleshooting-guide).
+* [Stack Overflow](https://stackoverflow.com/questions/tagged/uno-platform).
+* The [Uno gitter room](https://gitter.im/uno-platform/Lobby)
+* On [Twitter using the #unoplatform](https://twitter.com/search?q=%23unoplatform) hashtag.
+
+Organizations that want a deeper level of support beyond community support, should be directed to [contact nventive](https://platform.uno/contact/) to discuss obaining professional support.
+
+
+## Validate if the issue is a bug
+
+Confirm if the problem is a bug by reproducing it. If a test case has not been supplied ask the reporter to supply one. If reproducible, move to the next step of defining priority. You may need to contact the issue reporter in the following cases:
+
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know it by marking it duplicate. Label such issues as `triage/duplicate`.
+* If you can not reproduce the issue, label it as a `triage/not-reproducible`. Contact the issue reporter with your findings and close the issue if both the parties agree that it could not be reproduced.
+* If you need more information to further work on the issue, let the reporter know it by adding an issue comment followed by label `triage/needs-information`.
+
+In all cases, if you do not get a response in 20 days then close the issue with an appropriate comment.
+
+## Define priority
+
+We use GitHub issue labels for prioritization. The absence of a priority label means the bug has not been reviewed and prioritized yet.
+
+We try to apply these priority labels consistently across the entire project, but if you notice an issue that you believe to be incorrectly prioritized, please do let us know and we will evaluate your counter-proposal.
+
+If you want an issue resolved faster, please tell the maintainers that you are willing to send in a pull-request and we'll do everything we can to bring you to speed with becoming a contributor.
+
+* `priority/critical-urgent`: Must be actively worked on as someone's top priority right now. Stuff is burning. If it's not being actively worked on, someone is expected to drop what they're doing immediately to work on it. Team leaders are responsible for making sure that all the issues, labeled with this priority, are being actively worked on. Examples include user-visible bugs in core features, broken builds or tests and critical security issues.
```

Comment:
> user-visible bugs in core features 

I'm not sure what this means. I don't know which ones are the 'core features' and (nearly) all bugs are user-visible. I think the other two criteria (broken builds, security vulnerabilities) are valid.

Reply:
This document was based off the guidelines authored by the Kubernetes community sig:
https://github.com/kubernetes/community/blob/e4eabb7c3c26ce4463f1ab89e0ad94a7a7671b08/contributors/guide/issue-triage.md and `core features` comes from there. It has been changed to `time-sensitive hotfixes`